frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -102,19 +102,19 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
   };
 
   return (
-    <Tooltip
-      role="none"
+    <Popover
       data-testid="model-status-tooltip"
-      content={
+      position="top"
+      bodyContent={
         modelStatus?.failedToSchedule ? (
           <Text>Insufficient resources</Text>
         ) : (
           <Text>{getInferenceServiceStatusMessage(inferenceService)}</Text>
```

Comment:
Not a blocker but currently this causes an empty popover to render for the pending states. Wondering if we can add some logic here to prevent rendering a popover in those cases? This was an existing bug with the popovers as well.

Before:
<img width="98" alt="Screenshot 2024-10-17 at 4 27 11 PM" src="https://github.com/user-attachments/assets/1587f9a7-7b7c-4202-ad7f-a8d371e45c76">

After:
<img width="120" alt="Screenshot 2024-10-17 at 4 23 55 PM" src="https://github.com/user-attachments/assets/27ae448e-0df6-40dd-ac03-285466b4d07b">

Reply:
The UX has some bigger change requests for the contents.
For now I suggest to simply pre-compute the body content and then omit the popover or control its visibility by `isVisible={bodyContent ? undefined : false}`.